README.md

```diff
@@ -42,27 +42,51 @@ benchmark [arguments (optional)]
 * __Method:__ `autocannon -c 100 -d 5 -p 10 localhost:3000` (two rounds; one warm-up, one to measure).
 * __Node:__ `8.11.2`
 
-&nbsp;                   | Version | Router | Requests/s | Latency | Throughput/Mb
------------------------- | ------- | :----: | ---------: | ------: | ------------:
-bare                     | 8.11.2  | ✗      | 28411.2    | 3.44    | 4.04
-connect-router           | 1.3.2   | ✓      | 30328      | 3.22    | 4.36
-connect                  | 3.6.6   | ✗      | 32076.8    | 3.04    | 4.53
-express-route-prefix     | 4.16.2  | ✓      | 14447.2    | 6.81    | 4.97
-express-with-middlewares | 4.16.2  | ✓      | 14988      | 6.53    | 5.42
-express                  | 4.16.2  | ✓      | 20776      | 4.7     | 3.24
-fastify-big-json         | 1.5.0   | ✓      | 5002.8     | 19.43   | 57.40
-**fastify**              | **1.5.0** | ✓    | **33924.81** | **2.87** | **5.33**
-hapi                     | 17.5.0  | ✓      | 20116.8    | 4.88    | 3.11
-koa-router               | 7.4.0   | ✓      | 22059.2    | 4.44    | 3.44
-koa                      | 2.5.0   | ✗      | 25364.8    | 3.86    | 3.99
-micro                    | 9.1.0   | ✗      | 31902.4    | 3.06    | 4.97
-microrouter              | 3.1.1   | ✓      | 17989.6    | 5.44    | 2.84
-polka                    | 0.4.0   | ✓      | 34177.6    | 2.85    | 3.79
-rayo                     | 0.5.8   | ✓      | 34238.4    | 2.85    | 3.84
-restify                  | 7.1.0   | ✓      | 23956.8    | 3.89    | 3.84
-spirit-router            | 0.5.0   | ✓      | 29272      | 3.16    | 4.58
-spirit                   | 0.6.1   | ✗      | 30785.6    | 3.01    | 4.83
-take-five                | 1.3.4   | ✓      | 24491.2    | 3.7     | 8.07
-total.js                 | 2.9.4   | ✓      | 21019.2    | 4.65    | 6.03
-trek-engine              | 1.0.5   | ✗      | 28718.4    | 3.4     | 4.06
-trek-router              | 1.2.0   | ✓      | 28513.6    | 3.43    | 3.99
+
```

Comment:
@mcollina -Since this is not a markdown table, it looks pretty [wonky](https://github.com/fastify/benchmarks/blob/be0e7406f4d6fafb435df5d0cc4af3884913eb7f/README.md)...

Reply:
that's what `benchmarks.js compare -t` outputs.

Reply:
Yep, but that's a "command line table", not copy-n-paste compatible with markdown. I can update the code to pump out a markdown table, if that's more what you intented.
It's easy though, just wrap that table in \`\`\` and it will look perfectly pretty.

Reply:
We need something copy-pastable. I would prefer a markdown table, but I'll just do fine with ``` for now.